src/main/java/com/hillel/items_exchange/service/ImageService.java

```diff
@@ -70,4 +70,12 @@
      * @param imageId ID image to remove
      */
     void removeById(long imageId);
+
+    /**
+     * Makes scaled image (thumbnail) from given image bytes
+     * @param bytes bytes representative of image to scale
```

Comment:
Double word 'bytes' is it OK?

Reply:
The first word is the name of parameter, the second one is the start of description

Reply:
Ok)